README.md

```diff
@@ -64,15 +64,26 @@ Also check out [@ProjectAntrea](https://twitter.com/ProjectAntrea) on Twitter!
 
 ## Features
 
-Antrea currently supports the following features:
-* IPv4 overlay network for a Kubernetes cluster. Geneve, VXLAN, GRE, or STT can
-be used as the encapsulation protocol.
-* [Kubernetes Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies)
-implementation.
-* [Octant](https://github.com/vmware-tanzu/octant) UI plugin for monitoring
-Antrea components, which publish runtime information as
-[CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
-* [IPsec encyption](/docs/ipsec-tunnel.md) of GRE tunnel traffic.
+* **Kubernetes-native**: Antrea follows best practices to extend the Kubernetes
+  APIs and provide familiar abstractions to users, while also leveraging
+  Kubernetes libraries in its own implementation.
+* **Run everywhere**: Multiple traffic modes (with or without overlay) are
+  supported to ensure that Antrea can run in private clouds, public clouds and
+  at the edge.
```

Comment:
By edge, do we imply ARM here? The term is a little vague to me.

Reply:
I feel traffic modes are not directly relevant to platforms. Should we say: "Antrea can run in private clouds, public clouds, edge with multiple traffic modes"?

Reply:
I will replace "edge" with "bare metal", since edge tends to imply ARM for which we don't have a build yet

regarding the second comment, I will try to rephrase but I don't want to mislead either: I feel like as things stand today, the platform more or less dictates which traffic mode you will use

Reply:
So maybe: "Run Antrea in private clouds, public clouds and on bare metal, and select the appropriate traffic mode (with or without overlay) based on your platform and use case"

Reply:
Sounds good to me.